tests/misc_env/install_prereq.py

```diff
@@ -141,18 +144,35 @@ def setup_subscription_manager(ceph, timeout=1800):
     )
     while True:
         try:
-            ceph.exec_command(
-                cmd="sudo subscription-manager --force register  "
-                "--serverurl=subscription.rhsm.stage.redhat.com:443/subscription  "
-                "--baseurl=https://cdn.redhat.com --username=rhcsuser --password=rhcsuser",
-                timeout=720,
-            )
+            command = "sudo subscription-manager --force register --username={username} --password={password} "
+            if is_production:
+                credential = get_cephci_config("cdn_credentials")
+                command = command.format(
+                    username=credential["cdn_credentials"]["username"],
+                    password=credential["cdn_credentials"]["password"],
+                )
+
+            else:
+                credential = get_cephci_config("stage_credentials")
+                command += (
+                    "--serverurl=subscription.rhsm.stage.redhat.com:443/subscription "
+                    "--baseurl=https://cdn.redhat.com "
+                )
+                command = command.format(
+                    username=credential["stage_credentials"]["username"],
+                    password=credential["stage_credentials"]["password"],
+                )
+
+            ceph.exec_command(cmd=command, timeout=720)
 
             ceph.exec_command(
                 cmd="sudo subscription-manager attach --pool 8a99f9a471e63f73017212730c2c4fb6",
                 timeout=720,
             )
             break
+
```

Comment:
In conjunction with the above
```
except AttributeError:
  log.error("Require the %s credential", "cdn_credentials" if is_production else "stage_credentails")
```

Reply:
Thank you for the suggestions, I've made changes as suggested